### PR TITLE
Hotfix: render uiowa_hours via JS.

### PR DESCRIPTION
**Pull request by richardbporter**
Related to #4651 

Hotfix to get caching old hours resolved ASAP. The ideal scenario is probably what I described in #4651 as setting up a new route, controller, services, etc. and then making the AJAX call directly in JS instead of autosubmitting the form. 

# How to test

- `blt sync --site recserv.uiowa.edu`
- See the update hook runs.
- Visit building pages and see hours loaded via JS.
- Log in and verify that the hours form is not submitted while editing the page in Layout Builder and that placeholder text is visible instead.
- Double check the https://recserv.uiowa.ddev.site/hours page works correctly.

Also pushed to dev for Varnish testing: https://recserv.dev.drupal.uiowa.edu

### CODE REVIEW

**Comment by joewhitsitt** [comment]
I plan to take a look at this shortly.

**Review by joewhitsitt — APPROVED** [review]
looks better than a couple minutes ago! i'd like to keep working towards a more sustainable solution for dynamic blocks

File: docroot/modules/custom/uiowa_hours/src/Form/HoursFilterForm.php [hunk 1]
```diff
@@ -86,18 +90,35 @@ public function buildForm(array $form, FormStateInterface $form_state, $config =
...
+      '#type' => 'submit',
+      '#value' => $this->t('Submit'),
+      '#ajax' => [
+        'callback' => '::dateFilterCallback',
+        'wrapper' => 'results-container',
+        'disable-refocus' => TRUE,
+      ],
+      '#attributes' => [
+        'class' => [
+          'element-invisible',
```
Comment (richardbporter) [inline docroot/modules/custom/uiowa_hours/src/Form/HoursFilterForm.php:118]: @joewhitsitt is this the "right" thing to use here?

Comment (joewhitsitt) [reply docroot/modules/custom/uiowa_hours/src/Form/HoursFilterForm.php:118]: if the submit button is only there for the js to use, then it should probably be a display: none instead but i don't think we have a utility class for that available and i wonder if the js would still be able to act on it.

Comment (richardbporter) [reply docroot/modules/custom/uiowa_hours/src/Form/HoursFilterForm.php:118]: Did a quick non-JS test and things aren't working so I might just go with this for now. Thanks.

**Pull request merged**